### DISCUSSION TITLE
New Pulumi Package

### DISCUSSION THREAD

**Discussion by thatplatypus** [thomhurst/ModularPipelines#834] [Show and tell]
Hello! 

I just wanted to take a moment to say how much I love your library—it’s been a game-changer for me as I work to replace my clunky YAML pipelines with Modular Pipelines.

Since I use Pulumi for my CI/CD, I decided to create a small package to wrap their CLI tool. It’s open-sourced and available [here](https://github.com/Mythetech/ModularPipelines.Pulumi). Right now, it covers the essentials I needed, but I plan to expand support for more Pulumi commands over time.

I thought I’d share it here in case others might find it useful, and also to showcase how people are using and extending your fantastic library. 😄

Thanks again for all your hard work on this project—it’s incredibly appreciated! I’m really looking forward to fully switching everything over to Modular Pipelines.

**Comment by thomhurst** [comment]
Amazing, glad you're enjoying it @thatplatypus and thanks for sharing your lib! If you ever want to, happy to integrate it into the main repo so it could possible get more community support 😄
